tests/processor/test_tasks.py

```diff
@@ -693,3 +715,7 @@ def test_update_summary_tables_ocp_end_date(self):
 
         self.assertEqual(result_start_date, expected_start_date)
         self.assertEqual(result_end_date, expected_end_date)
+
+    def test_update_charge_info_aws(self):
```

Comment:
I think there's missing update charge task tests and/or something didn't get committed.

Reply:
Nope sounds good.